torchao/_models/llama/generate.py

```diff
@@ -189,21 +189,21 @@ def main(
     if quantization:
         from torchao.quantization.quant_api import (
```

Comment:
can we dedup the quant code in eval and generate.py?

Reply:
only a bit, its probably more trouble that its worth given the differences and needing to handle autoquant vs eval